src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -101,11 +101,11 @@ const MessageComposerButtons: React.FC<IProps> = (props: IProps) => {
 
     return <UploadButtonContextProvider roomId={roomId} relation={props.relation}>
         { mainButtons }
-        <AccessibleTooltipButton
+        { moreButtons.length != 0 && <AccessibleTooltipButton
```

Comment:
```suggestion
        { moreButtons.length > 0 && <AccessibleTooltipButton
```

Probably just my preference :sweat_smile: